app/controllers/awards_controller.rb

```diff
@@ -0,0 +1,32 @@
+class AwardsController < ApplicationController
+  before_action :require_admin, only: [:create]
+  before_action :set_award, only: [:show]
+
+  def index
+    @awards = Award.all
+    @award = Award.new
+  end
+
+  def show
+
+  end
+
+  def create
+    award = Award.new(award_params)
+    award.save
```

Comment:
these two lines can be combined in the future to just `award = Award.create(award_params)`